packages/web-components/src/components/card-group/__stories__/card-group.stories.ts

```diff
@@ -131,6 +132,58 @@ withImagesAndCTA.story = {
   },
 };
 
+export const withMixedMedia = () => {
+  return html`
+    <dds-video-cta-container>
+      <dds-card-group>
+        <dds-card-group-item cta-type="video" href="1_9h94wo6b">
+        <dds-card-eyebrow>Topic</dds-card-eyebrow>
+          Test Video - 1:1
+          <dds-card-cta-footer cta-type="video" slot="footer" href="1_9h94wo6b">
+            0:18
```

Comment:
Can we align the content models on these? Is there anything we can do to do that? It's odd because I see `eyebrow` `heading` and `cta` for video here, and the same in the default image version below, but when it renders on the page some of the elements are being removed.

![image](https://user-images.githubusercontent.com/3793636/111484572-e4e81a00-8703-11eb-911e-f81bf14f7a39.png)